clang 実行時エラー

```
MISO/demo/hd1d_shock_tube/src/main.cpp:2:
/Users/hotta/presentation/2026/中日文化センター/MISO/miso/include/miso/mhd_model_base.hpp:104:34: error: 
      use 'template' keyword to treat 'as' as a dependent template name
  104 |     if (config["io"]["continue"].as<bool>() &&
      |                                  ^
      |                                  template 
1 error generated.

```

<!-- この問題はリポジトリ is-isee/MISO (https://github.com/is-isee/MISO) に作成されます。この行を変更した場合、効果はありません。 -->

担当者:
ラベル: 
マイルス トーン: 
プロジェクト: 



<!-- 新しいイシューの本文を編集し、エディターの右上にある ✓ [イシューの作成] ボタンをクリックします。最初の行がイシューのタイトルになります。担当者とラベルは空白行の後に続きます。イシューの本文を開始する前に空の行を残します。 -->

--- a/miso/include/miso/mhd_model_base.hpp
+++ b/miso/include/miso/mhd_model_base.hpp
@@ -101,7 +101,7 @@ public:
                   "Derived must have member variable 'src'.");
 
     mhd.apply_initial_condition(d.ic, d.bc);
-    if (config["io"]["continue"].as<bool>() &&
+    if (config["io"]["continue"].template as<bool>() &&
         fs::exists(fs::path(time.time_save_dir) / "n_output.txt")) {
       load_state();
     }

